src/swagger-specs/destinations.yaml

```diff
@@ -34,81 +34,313 @@ servers:
+      operationId: getConnections
       parameters:
-        - in: header
-          name: Authorization
-          description: "The access token which can be copied from your Experience Platform integration, prefixed with 'Bearer '. For more information on how to obtain this value, visit the [authentication tutorial](http://www.adobe.com/go/platform-api-authentication-en)."
-          schema:
-            type: string
-          required: true
-        - in: header
-          name: Content-type
-          description: "The type of content being sent in the body of the request. Should be 'application/json'."
-          schema: 
-            type: string
-            default: "application/json"
-          required: true
-        - in: header
-          name: x-api-key
-          description: "The API key which can be copied from your Experience Platform integration. For more information on how to obtain this value, visit the [authentication tutorial](http://www.adobe.com/go/platform-api-authentication-en)."
-          schema:
-            type: string
-          required: true
-        - in: header
-          name: x-gw-ims-org-id
-          description: "The organization ID which can be copied from your Experience Platform integration. For more information on how to obtain this value, visit the [authentication tutorial](http://www.adobe.com/go/platform-api-authentication-en)."
+        - $ref: '#/components/parameters/Authorization'
+        - $ref: '#/components/parameters/XApiKey'
+        - $ref: '#/components/parameters/XImsOrgId'
+        - $ref: '#/components/parameters/XSandboxId'
+        - $ref: '#/components/parameters/Accept'
+        - in: query
+          name: property
+          description: > 
+            A comma-separated list of top-level object properties to be returned in the response. Used to cut down the amount
+            of data returned in the response body. For example, `property=id==f2d1a98b-6b8c-420b-80a1-696c3f192eb2,b6730e09-7da6-40e2-b5cf-289545056dc3,49d17eff-9fa7-4e92-b84e-ab65fcb4730a`.
+          required: false
           schema:
             type: string
+          examples:
+            filterS3Connection:
+              summary: Filter to return connections for Amazon S3 destinations only.
+              value: "connectionSpec.id==4890fc95-5a1f-4983-94bb-e060c08e3f81"
+            filterSftpConnection:
+              summary: Filter to return connections for SFTP destinations only.
+              value: "connectionSpec.id==64ef4b8b-a6e0-41b5-9677-3805d1ee5dd0"
+        - $ref: '#/components/parameters/limit'
+        - $ref: '#/components/parameters/orderby'       
+        - $ref: '#/components/parameters/count'
+        - $ref: '#/components/parameters/continuationToken' 
+
+      responses:
+        '200':
+          description: Successful operation
```

Comment:
The `description` for success responses should mention what is returned in the response at a glance, so the reader can get an idea before they click into the details. "A successful response returns a list of base connections that match the request parameters."